firefox-ios/Client/Frontend/Browser/FirefoxTabContentBlocker.swift

```diff
@@ -113,11 +114,28 @@ class FirefoxTabContentBlocker: TabContentBlocker, TabContentScript {
         )
     }
 
+    private func dispatchSafeListStatusAction(for blockerStatus: BlockerStatus?) {
+        guard featureFlags.isFeatureEnabled(.toolbarRefactor, checking: .buildOnly) else { return }
+
+        let safeListedURLImageName = (
+            blockerStatus == .safelisted
+        ) ? StandardImageIdentifiers.Small.notificationDotFill : nil
+
+        let action = ToolbarAction(
+            safeListedURLImageName: safeListedURLImageName,
+            windowUUID: .unavailable,
```

Comment:
If we dispatch without a `windowUUID` present all windows will react to this so this could result in a website being shown as safelisted that is not because we can have multiple windows on iPad.